pkg/sql/codegen/attribute/attribute.go

```diff
@@ -158,3 +159,35 @@ func (d Dictionary) GenerateTableInHTML() string {
 	l = append(l, `</table>`)
 	return strings.Join(l, "\n")
 }
+
+// Update updates `d` by copyint from `other` key by key
+func (d Dictionary) Update(other Dictionary) Dictionary {
+	for k, v := range other {
+		d[k] = v
+	}
+	return d
+}
+
+// NewDictionary create a new Dictionary according to `estimator`
+func NewDictionary(estimator, prefix string) Dictionary {
```

Comment:
`estimator` is Tensorflow specific, and we should not add engine specific code in `attribute.go`

Reply:
`estimator` is not TensorFlow specific, sklearn proposed the original concept and it is widely accepted by the AI community, XGBoost implemented the `estimator` interface too.
For another example, we also have `ir.TrainClause.Estimator` with the same meaning as the `estimator` here.